lib_nn/api/Conv2d.hpp

```diff
@@ -0,0 +1,64 @@
+#include <cmath>
+#include <tuple>
```

Comment:
I might have missed it. We don't seem to be using tuple. Could you please remove if not using it.